### ISSUE TITLE
Loss fluctuates

### ISSUE THREAD

**Issue by Emrys-Hong** [Emrys-Hong/sutd-machine-learning-project#3]
For MEMM/CRF, because of the strict convexity of its loss function, the loss should do down?

but in training if set the learning rate to be high enough, the loss will fluctuates although the model converges
